content/en/docs/concepts/services-networking/service.md

```diff
@@ -1352,211 +1177,26 @@ spec:
     - 80.11.12.10
 ```
 
-## Shortcomings
-
-Using the userspace proxy for VIPs works at small to medium scale, but will
-not scale to very large clusters with thousands of Services.  The
-[original design proposal for portals](https://github.com/kubernetes/kubernetes/issues/1107)
-has more details on this.
-
-Using the userspace proxy obscures the source IP address of a packet accessing
-a Service.
-This makes some kinds of network filtering (firewalling) impossible.  The iptables
-proxy mode does not
-obscure in-cluster source IPs, but it does still impact clients coming through
-a load balancer or node-port.
-
-The `Type` field is designed as nested functionality - each level adds to the
-previous.  This is not strictly required on all cloud providers (e.g. Google Compute Engine does
-not need to allocate a `NodePort` to make `LoadBalancer` work, but AWS does)
-but the Kubernetes API design for Service requires it anyway.
```

Comment:
This paragraph was deleted intentionally?

Reply:
I think this paragraph is now misleading. However, the better place to remove it is PR #37385 so I will _move_ it in this PR and then deal with the rebase fun after whichever of those lands first.